src/Sql/Sql/Database/Cmdlet/SetAzureSqlDatabase.cs

```diff
@@ -176,6 +176,43 @@ public class SetAzureSqlDatabase : AzureSqlDatabaseCmdletBase<IEnumerable<AzureS
             Management.Sql.Models.DatabaseLicenseType.BasePrice)]
         public string LicenseType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the compute model for the Azure Sql Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Computed model of Azure Sql database, serverless or provisioned",
+            ParameterSetName = UpdateParameterSetName)]
+        [Parameter(Mandatory = false,
+            HelpMessage = "Computed model of Azure Sql database, serverless or provisioned",
+            ParameterSetName = VcoreDatabaseParameterSet)]
+        [PSArgumentCompleter(
+            "Provisioned",
+            "Serverless")]
+        public string ComputeModel { get; set; }
+
+        /// <summary>
+        /// Gets or sets the auto pause delay for the Azure Sql Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The auto pause delay for Azure Sql database (serverless only), -1 to opt out",
+            ParameterSetName = UpdateParameterSetName)]
+        [Parameter(Mandatory = false,
+            HelpMessage = "The auto pause delay for Azure Sql database (serverless only), -1 to opt out",
+            ParameterSetName = VcoreDatabaseParameterSet)]
+        public int? AutoPauseDelay { get; set; }
```

Comment:
Should use `TimeSpan` or have units in the name

Reply:
This is the same name as we have in API, so it can be confusing to actually have different name here.

Reply:
99% of powershell users don't look at the API. The name should have units in it. Please change to AutoPauseDelayMinutes.

Reply:
Renamed to AutoPauseDelayMinutes